specs/TransciptTestRunner.md

```diff
@@ -37,6 +40,59 @@ Here is a high level class diagram to get started
 - `_XYZ_Client`: specialized client that knows how to interact with a specific channel (it should be capable of generating channel OAuthTokens, handle responses coming from that specific channel, etc.)
 - `ClientType` (enum): types are: DirectLine, Emulator, Teams
 
+## TestRunner Class
+### Methods
+- `RunTestAsync(string)`  Executes a single scenario based on the transcript passed to the method.   
```

Comment:
I'm thinking we may want to rename this to `RunScriptAsync` or `RunTranscriptAsync` something different.